tests/common.go

```diff
@@ -12129,3 +12136,12 @@ func RefreshIscsiSession(n node.Node) error {
 	}
 	return nil
 }
+
+// GetPVCObjFromVol Returns pvc object from Volume
+func GetPVCObjFromVol(vol *volume.Volume) (*v1.PersistentVolumeClaim, error) {
+	pvcObj, err := k8sCore.GetPersistentVolumeClaim(vol.Name, vol.Namespace)
+	if err != nil {
```

Comment:
Minor comment - we can simply return pvcObj, err here as you are anyways doing log.FailOnError in the test case so this condition is being checked there

Reply:
Done